concrete/single_pages/dashboard/system/seo/urls.php

```diff
@@ -66,14 +66,14 @@
     $(function () {
 
         var steps = [{
-            content: '<p><span class="h5"><?=t('Pretty URLs')?></span><br/><?=t('Check this checkbox to remove index.php from your URLs. You will be given code to place in a file named .htaccess in your web root. concrete5 will try and place this code in the file for you.')?></p>',
+            content: <?= json_encode('<p><span class="h5">' . t('Pretty URLs') . '</span><br/>' . t('Check this checkbox to remove index.php from your URLs. You will be given code to place in a file named .htaccess in your web root. Concrete5 will try and place this code in the file for you.') . '</p>') ?>,
```

Comment:
Thanks, I hadn't noticed that change. I didn't try to "correct" that, it was copy-pasted from an also incorrect source file.